test/vtcomposite-points.test.js

```diff
@@ -49,13 +49,16 @@ test('[composite] success overzooming - different zooms between two tiles, no bu
   const info = vtinfo(buffer1);
   assert.equal(info.layers.quadrants.length, 4);
   const originalGeometry = info.layers.quadrants.feature(0).loadGeometry()[0][0];
-
+  console.log('quads!!', info.layers.quadrants.feature(0).loadGeometry());
```

Comment:
Did you mean to commit this?

Reply:
ehhh nope. sorry wasn't quite ready for a review yet, but your comments are much appreciated! :)